FogCreek/Glitch-Community
# Sign-In-Updates: Updated styling of login pages, fixed create team pop-over styling

## Links
* Remix link: https://sign-in-updates.glitch.me/
* Issue-tracker link: 
https://app.clubhouse.io/glitch/story/4511/update-sign-in-styling
https://app.clubhouse.io/glitch/story/4280/create-team-pop-width-incorrect

## GIF/Screenshots:
![image](https://user-images.githubusercontent.com/519336/69274119-b3128200-0ba7-11ea-9fcd-aa0522c2c799.png)
* Edited oauth button ordering to put Facebook last
* made button size of login items consistent
* increased margin around terms and conditions

![image](https://user-images.githubusercontent.com/519336/69273797-10f29a00-0ba7-11ea-9a4e-89faa28e0d5f.png)
* fixed pop-over title text size

![image](https://user-images.githubusercontent.com/519336/69273880-397a9400-0ba7-11ea-9330-43a076914df4.png)
Fixed teams pop-over width

## How To Test:
* https://sign-in-updates.glitch.me/login - check that things look consistent with the changes described above
* https://sign-in-updates.glitch.me/ - log in and check out the create-teams-pop
